artifactory/commands/golang/go.go

```diff
@@ -19,24 +15,39 @@ import (
 	"github.com/jfrog/jfrog-client-go/auth"
 	"github.com/jfrog/jfrog-client-go/utils/errorutils"
 	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
 	"github.com/jfrog/jfrog-client-go/utils/version"
+	"path"
+	"path/filepath"
+	"strings"
 )
 
-const GoCommandName = "rt_go"
-
 type GoCommand struct {
 	noRegistry         bool
 	publishDeps        bool
```

Comment:
Let's also remove the above ```publishDeps``` and ```noRegistry``` options and add them to the doc describing the jfrog-cli v2 changes.